isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -2109,6 +2230,109 @@ namespace isobus
 		}
 	}
 
+	void VirtualTerminalClient::process_esc_message_callback(std::uint16_t objectID, ESCMessageErrorCode errorCode, VirtualTerminalClient *parentPointer)
+	{
+		for (uint32_t i = 0; i < parentPointer->escMessageCallbacks.size(); i++)
```

Comment:
Nit: `size()` returns `size_t` so we should probably use `std::size_t` in these rather than `uint32_t`. Prefer to have `std::` explicitly